### PR TITLE
Fix ci_scripts args management

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date.

### CODE REVIEW

**Review by rebtoor — APPROVED** [review]

File: plugins/action/ci_script.py [hunk 1]
```diff
@@ -122,6 +122,13 @@
 
 
 class ActionModule(ActionBase):
+    __OUTPUT_DIR_ARG = "output_dir"
+    __SCRIPT_ARG = "script"
+    __DRY_RUN_ARG = "dry_run"
+    __EXTRA_ARGS_ARG = "extra_args"
+    __CHDIR_ARG = "extra_args"
```
Comment (cjeanner) [inline plugins/action/ci_script.py:129]: I think it should be:
```suggestion
    __CHDIR_ARG = "chdir"
```

File: plugins/action/ci_script.py [hunk 2]
```diff
@@ -154,38 +161,40 @@ def run(self, tmp=None, task_vars=None):
...
+        file_task.args.pop(self.__CHDIR_ARG, None)
 
-        # Are we running dry-run?
-        dry_run = False
-        if "dry_run" in task_args:
-            dry_run = basic.boolean(task_args.pop("dry_run"))
+        # Remove custom options from our plugin that are not part of Ansible's one
+        file_task.args.pop(self.__OUTPUT_DIR_ARG, None)
+        file_task.args.pop(self.__DRY_RUN_ARG, None)
+        file_task.args.pop(self.__EXTRA_ARGS_ARG, None)
```
Comment (cjeanner) [inline plugins/action/ci_script.py:186]: please also `pop(self.__SCRIPT_ARG, None)`

**Comment by cjeanner** [comment]
/approve

current iteration works for me, thanks!

**Comment by cjeanner** [comment]
Prow cluster is dead, manually merging to unblock users.

**Pull request merged**